contracts/token/IToken.sol

```diff
@@ -0,0 +1,15 @@
+pragma solidity ^0.5.10;
+
+//interface
+contract IToken{
+
+    string public name;
+    string public symbol;
+    uint8 public constant decimals = 0;
```

Comment:
i don't think we should set a value for this on the interface, it should be set on the implementation side.

Reply:
https://github.com/TokenySolutions/T-REX/pull/17